src/definitions/h.json

```diff
@@ -0,0 +1,125 @@
+[
+  {  
+      "display": "Place the punctuation",
+      "description": "The hanging-punctuation property specifies whether a punctuation mark may be placed outside the line box at the start or at the end of a full line of text.",
+      "link": "https://www.w3schools.com/cssref/css3_pr_hanging-punctuation.asp",
+      "propertyName": "hanging-punctuation",
+      "valuetemplate": "{0}",
+      "categories": ["misc"],
+      "tags": "punctuation, character, paragraph",
+      "controls": [
+        {
+          "type": "select",
+          "default": "none",
+          "label": "Value",
+          "options": [
+            {
+              "name": "No Punctuation",
+              "value": "none"
+            },
+            {
+              "name": "At the First",
+              "value": "first"
+            },
+            {
+              "name": "At the Last",
+              "value": "last"
+            },
+            {
+              "name": "Allow at the End",
+              "value": "allow-end"
+            },
+            {
+              "name": "Force at the End",
+              "value": "force-end"
+            },
+            {
+              "name": "Default Value",
+              "value": "initial"
+            },
+            {
+              "name": "Inherit from parent",
+              "value": "inherit"
+            }
+          ]
+        }
+      ]
+    },
+    {  
+        "display": "Set the height",
+        "description": "The height property sets the height of an element. The height of an element does not include padding, borders, or margins!",
+        "link": "https://www.w3schools.com/cssref/pr_dim_height.asp",
+        "propertyName": "height",
+        "valuetemplate": "{0}",
+        "categories": ["misc", "layout", "container"],
+        "tags": "size, height, scale",
+        "controls": [
+          {
+            "type": "select",
+            "default": "auto",
+            "label": "Value",
+            "options": [
+              {
+                "name": "Auto",
+                "value": "auto"
+              },
+              {
+                "name": "Set in pixel(px)",
```

Comment:
As height can be set as "px" or "%", let the user decide what unit to use along with the value. in another word, there should be an option called "Custom" which will allow the user to enter the value in  <value><unit> format
Refer border-radius

Reply:
So we want it like a select box with values, default(initial), auto. inherit, and custom? When custom selected, we get a range and another selector to select px, %, etc?
Is this kind of relationship possible by framework now?